connector/src/main/resources/initial_data.json

```diff
@@ -184,5 +184,22 @@
       "modified": "2024-04-23T16:26:00.000Z",
       "version": 0
     }
+  ],
+  "application_properties": [
```

Comment:
Instead of the test properties, we should put here real-world ones, such are the DAPS, for which you included the logic